apps/reactotron-app/src/renderer/KeybindHandler.tsx

```diff
@@ -127,13 +134,22 @@ function KeybindHandler({ children }) {
     ToggleSidebar: () => {
       toggleSideBar()
     },
+    ToggleSearch: () => {
+      // If we're on the timeline page, toggle the search, otherwise switch to the timeline tab and open search
+      if (window.location.hash === "#/") {
+        toggleSearch()
+      } else {
+        openSearch()
+        handlers.OpenTimelineTab()
+      }
+    },
     ClearTimeline: () => {
       clearCommands()
     },
   }
 
   return (
-    <GlobalHotKeys keyMap={keyMap as any} handlers={handlers}>
+    <GlobalHotKeys keyMap={keyMap as any} handlers={handlers} allowChanges>
```

Comment:
[`allowChanges`](https://www.npmjs.com/package/react-hotkeys#allowing-hotkeys-and-handlers-props-to-change)

allows for the handlers to update accordingly instead of always being memoized. functions, like `toggleSearch` need to update accordingly to any related prop changes to actually toggle.